src/Forms/DateField.php

```diff
@@ -536,13 +453,12 @@ protected function frontendToInternal($date)
      * as defined by {@link $dateFormat}. With $html5=true, the frontend date will also be
      * in ISO 8601.
      *
-     * @param string $date
      * @return string The formatted date, or null if not a valid date
      */
-    protected function internalToFrontend($date)
+    public function internalToFrontend(mixed $value): ?string
```

Comment:
We probably shouldn't change named parameters without good cause - `$date` is a perfectly good name for this parameter.
Every change to the API adds upgrade pain so we shouldn't be making unnecessary changes.
I won't block merging for this, it's more of a strong recommendation.
```suggestion
    public function internalToFrontend(mixed $date): ?string
```

Reply:
I've added an interfaces for the 'converter' functionality which has this

```php
interface FieldValidatorConverterInterface
{
    public function internalToFrontend(mixed $value): ?string;
```

So I think in this case we want to change the param name